tests/e2e/specs/editor/media/insertMediaFromLibrary.js

```diff
@@ -31,8 +31,11 @@ describe('Inserting Media from Media Library', () => {
 
     await expect(page).not.toMatchElement('[data-testid="FrameElement"]');
 
+    await page.waitForSelector('.mediaElementimage');
     // Clicking will only act on the first element.
-    await expect(page).toClick('[data-testid="mediaElement"]');
+    await expect(page).toClick('.mediaElementimage');
```

Comment:
Was the `waitForSelector` needed? `toClick` usually waits as well

Reply:
Just being overly careful here